### DISCUSSION TITLE
GET api/builds/{id}

### DISCUSSION THREAD

**Discussion by alexniculae** [Visual-Regression-Tracker/Visual-Regression-Tracker#358] [Q&A] [answered]
When making the following http request:
`GET api/builds/{id}`

Will the `data.isRunning` switch to `false`, as soon as the **vrtStop** command is triggered?

If there are multiple specs that trigger **vrtStart** and **vrtStop** iteratively, will the build still consider `isRunning === true`? If so, when does it change to `false`, when nothing is actively working on the build?

I am trying to understand if there is any chance of having `data.isRunning === false` after the last spec has ran and the **vrtStop** command was triggered for the last time.

**Comment by pashidlos** [comment, marked as answer]
thanks for your question
```
Will the data.isRunning switch to false, as soon as the vrtStop command is triggered?
```
yes, it will

```
If there are multiple specs that trigger vrtStart and vrtStop iteratively, will the build still consider isRunning === true? If so, when does it change to false, when nothing is actively working on the build?
```
currently `data.isRunning` is only responsible for visual representation on ui - loader 
so you could reuse the build even if it's stopped 

```
I am trying to understand if there is any chance of having data.isRunning === false after the last spec has ran and the vrtStop command was triggered for the last time.
```
invoke `vrtStop` inside global teardown after the last spec
assume you are using cypress, where global teardown is an issue 
call it after each spec as workaround
